src/script/lua_api/l_mapgen.cpp

```diff
@@ -1746,6 +1746,84 @@ int ModApiMapgen::l_serialize_schematic(lua_State *L)
 	return 1;
 }
 
+// read_schematic(schematic, options={...})
+int ModApiMapgen::l_read_schematic(lua_State *L)
```

Comment:
`deserialize_schematic` for consistency with `minetest.deserialize`

Reply:
I believe it could be confusing: `serialize_schematic` takes a Schematic specifier as input and returns a string (it serializes the schematic), but `read_schematic` *also* takes a Schematic specifier as input, but returns a table instead. It is not to `serialize_schematic` what `minetest.deserialize` is to `minetest.serialize`.